hawc/apps/vocab/constants.py

```diff
@@ -70,3 +70,15 @@ class Ontology(IntegerChoices):
     """
 
     umls = 1, "UMLS"
+
+
+class ObservationStatus(TextChoices):
+    """
+    Guideline profile observation status
+    """
+
+    NM = "NM", "NM"
+    NR = "not required", "not required"
+    REC = "recommended", "recommended"
+    REQ = "required", "required"
+    TR = "triggered", "triggered"
```

Comment:
Added type_checking for GuidelineProfile observation status, can be changed/removed as needed